### PR TITLE
Like messages from a notification

### PR DESCRIPTION
**Pull request by daehn**
# What's in this PR?

* Bump dependencies to enable liking form notifications.
* Update for new data-model API.

### This PR depends on:
- [x] A new release of `wire-ios-data-mode` after https://github.com/wireapp/wire-ios-data-model/pull/191 has been merged.
- [x] A new release of `wire-ios-sync-engine` after https://github.com/wireapp/wire-ios-sync-engine/pull/277 has been merged.

### CODE REVIEW

File: Wire-iOS/Sources/Helpers/syncengine/ZMMessage+Likes.swift [hunk 1]
```diff
@@ -20,10 +20,6 @@
 import Foundation
 import zmessaging
 
-public enum ZMMessageReaction: String {
-    case Like = "❤️"
-}
-
 extension ZMConversationMessage {
```
Comment (mikeger) [inline Wire-iOS/Sources/Helpers/syncengine/ZMMessage+Likes.swift]: I think this whole extension can move to `wire-ios-data-model`

Comment (daehn) [inline Wire-iOS/Sources/Helpers/syncengine/ZMMessage+Likes.swift]: We don't have the `Message` type and it's functions there, but yeah we could just check the actual message there as well.

Comment (daehn) [inline Wire-iOS/Sources/Helpers/syncengine/ZMMessage+Likes.swift]: Which on the other hand means we either duplicate the meaning of `isNormalMessage` or we move this logic into the data-model as well.

Comment (daehn) [inline Wire-iOS/Sources/Helpers/syncengine/ZMMessage+Likes.swift]: @mikeger I would suggest doing this general clean-up separately as this PR blocks other ones at the moment.

Comment (mikeger) [inline Wire-iOS/Sources/Helpers/syncengine/ZMMessage+Likes.swift]: Sure

**Comment by daehn** [comment]
@mikeger updated!

**Pull request merged**